pkg/utils/label/label_test.go

```diff
@@ -0,0 +1,66 @@
+package labels
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestHasLabel(t *testing.T) {
+	tests := []struct {
+		name   string
+		inp    *appsv1.Deployment
```

Comment:
Yes this can be done. Also, the `HasLabel` func should then take `metav1.Object` instead of `runtime.Object`.  `metav1.Object` doesn't implement  all the methods of`runtime.Object` so they're incompatible types.

Reply:
I've made the change. Thanks!